src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <section class="calendar calendar__start-day--tue calendar__month-length--30">
```

Comment:
better use modifiers of the calendar ("--")
```suggestion
    <section class="calendar calendar--start-day--tue calendar--month-length--30">
```